### PR TITLE
Add Support for Q.VOLT HYB-G3-3P

### PR DESCRIPTION
**Pull request by VadimKraus**
The "Q.VOLT HYB-G3-3P" is part of the "[Q.HOME+ ESS HYB-G3-3P](https://www.q-cells.de/installateure/qhome-ess-hyb-g3.html)" Set, which is a clone/relabel of  [X ESS G4](https://www.solaxpower.com/x-ess-g4/) and contains a "X3 Hybrid G4".

Since I only have access to the QCells variante, this PR adds an inverter with the name "QVOLTHYBG33P". 
The mapping was done by best effort, however, there are still some data points in the repose which I could not identify.

### CODE REVIEW

**Comment by squishykid** [comment]
I think it makes sense to add it to the library. We might want to split it out into a separate discovery mechanism though, so that home-assistant users don't get confused. For now we can just add it here and cross that bridge when they do get confused :)

File: solax/utils.py [hunk 1]
```diff
@@ -1,3 +1,25 @@
+INVERTER_OPERATION_MODES = {
```
Comment (squishykid) [inline solax/utils.py:1]: My only comment would be: Could you please move these added constants and functions inside the `QVOLTHYBG33P` class? The new class is the only class which currently uses these values.

**Review by squishykid — CHANGES_REQUESTED** [review]

**Comment by squishykid** [comment]
Should just be a small change and then it's ready to merge!

File: solax/utils.py (hunk 1 above)
Comment (VadimKraus) [reply solax/utils.py]: Like that?

**Comment by squishykid** [comment]
Nice work! Thanks for your contribution!

**Pull request merged**